source/Analysis/src/AnalyzerOccupancy.cc

```diff
@@ -91,29 +95,53 @@ AnalyzerOccupancy::~AnalyzerOccupancy()
   if (m_chargedMapBOffMatOff !=nullptr) delete m_chargedMapBOffMatOff;
 }
 
+//! Init variables
+bool AnalyzerOccupancy::init(int nTracks)
+{
+  std::string directory = MainConfigHandler::getInstance().getIrradiationDirectory();
+
+  bool bFieldMapOK  = checkFile(SimParms::getInstance().bFieldMapFile(), directory);
+  bool chargedMapOK = checkFile(SimParms::getInstance().chargedMapFile(), directory);
+  bool photonsMapOK = checkFile(SimParms::getInstance().photonsMapFile(), directory);
+
+  if (bFieldMapOK)  m_bFieldMap           = new BFieldMap(directory+"/"+SimParms::getInstance().bFieldMapFile());
+  if (chargedMapOK) m_chargedMapBOnMatOn  = new IrradiationMap(directory + "/" + SimParms::getInstance().chargedMapFile());
+  if (photonsMapOK) m_photonsMapBOnMatOn  = new IrradiationMap(directory + "/" + SimParms::getInstance().photonsMapFile());
+
+  m_isInitOK = bFieldMapOK && chargedMapOK && photonsMapOK;
+
+  return m_isInitOK;
+}
+
 bool AnalyzerOccupancy::analyze()
 {
+  // Check that initialization OK
+  if (!m_isInitOK) return false;
+
   // Make & fill all flux histograms
   fillHistogram(m_chargedMapBOnMatOn,   m_hisChargedFluxBOnMatOn,   "ChargedFluxPerPPBOnMatOn",   "Flux of charged particles [cm^{-2}] per pp collision - B on, all material on");
-  fillHistogram(m_chargedMapBOnMatOnLTh,m_hisChargedFluxBOnMatOnLTh,"ChargedFluxPerPPBOnMatOnLTh","Flux of charged particles [cm^{-2}] per pp collision - B on, all material on (e-low thr.)");
-  fillHistogram(m_chargedMapBOffMatOn,  m_hisChargedFluxBOffMatOn,  "ChargedFluxPerPPBOffMatOn",  "Flux of charged particles [cm^{-2}] per pp collision - B off, all material on");
-  fillHistogram(m_chargedMapBOnMatOff,  m_hisChargedFluxBOnMatOff,  "ChargedFluxPerPPBOnMatOff",  "Flux of charged particles [cm^{-2}] per pp collision - B on, material off");
-  fillHistogram(m_chargedMapBOffMatOff, m_hisChargedFluxBOffMatOff, "ChargedFluxPerPPBOffMatOff", "Flux of charged particles [cm^{-2}] per pp collision - B off, material off");
-  fillHistogram(m_chargedMapBOffTrkOff, m_hisChargedFluxBOffTrkOff, "ChargedFluxPerPPBOffTrkOff", "Flux of charged particles [cm^{-2}] per pp collision - B off, tracker material off");
+//  fillHistogram(m_chargedMapBOnMatOnLTh,m_hisChargedFluxBOnMatOnLTh,"ChargedFluxPerPPBOnMatOnLTh","Flux of charged particles [cm^{-2}] per pp collision - B on, all material on (e-low thr.)");
```

Comment:
I see some 'dead code' in AnalyzerOccupancy. Code that's commented out and left there. Is it possible to remove it? If not, could you please add a comment explaining if/when it should be to uncommented?